### ISSUE TITLE
Remove KireiCake?

### ISSUE THREAD

**Issue by Thermospore** [FooSoft/yomichan#1608]
I'm not sure the version currently on the yomichan info page serves much use
1. it mostly consists of an old copy of jmdict
2. the non-jmdict entries are of very low quality

To see the entries that didn't come from jmdict, look at the upper table here: https://kireicake.com/rikaicake-online/

I get the impression the creator made many of these entries on the spot when seeing the word for the first time

A given entry usually has a handful of these qualities:
1. straight up wrong
2. not entirely wrong, but not particularly accurate
3. very incomplete
4. has typos / formatting errors / inconsistencies
5. vastly inferior to the jmdict entry
6. vastly inferior to the jj entries
7. very apparent / unneeded A+B

some entries off the top of my head
![image](https://user-images.githubusercontent.com/35350819/114258817-bca3b200-9975-11eb-8cdb-c1d1ca86db93.png)
![image](https://user-images.githubusercontent.com/35350819/114258826-c9c0a100-9975-11eb-9668-0a5425f5bcf5.png)
![image](https://user-images.githubusercontent.com/35350819/114258810-b0b7f000-9975-11eb-8945-f03ed537ca41.png)
![image](https://user-images.githubusercontent.com/35350819/114258792-97af3f00-9975-11eb-8099-353c44a00d18.png)
![image](https://user-images.githubusercontent.com/35350819/114258799-a0a01080-9975-11eb-8cff-c3ee3ce0f8fd.png)
![image](https://user-images.githubusercontent.com/35350819/114258891-66833e80-9976-11eb-85e4-71ca4e3e500e.png)
![image](https://user-images.githubusercontent.com/35350819/114258805-a7c71e80-9975-11eb-94ed-c7be856bab78.png)

I think if it were to be kept on the yomichan page, the old jmdict entries should be removed at least. Here are builds I made from the original tables. I did my best to fix the formatting errors / inconsistencies. Beware if you want to make your own builds, as the upper table is particularly convoluted and inconsistent

[RikaiCake [upper table] v1.zip](https://github.com/FooSoft/yomichan/files/6289276/RikaiCake.upper.table.v1.zip)
[RikaiCake [lower table] v4.zip](https://github.com/FooSoft/yomichan/files/6289273/RikaiCake.lower.table.v4.zip)

I mean no offense to the creator, they obviously provide the tables in good will. But in general, even if it has something not in any of your other dictionaries you are probably better off googling it yourself

**Comment by FooSoft** [comment]
@Thermospore very good work. Do you mind if I put these on the download page instead of the old version that's there? I completely agree that the current KireiCake dictionary is a huge mess, and this looks to be a solid improvement.

**Comment by Thermospore** [comment]
@FooSoft Sure, you are welcome to use them. @toasted-nutbread might want to take a look at them first though to see if there are any changes they'd like

Each dictionary's metadata includes a link to the google sheet I used to build it, which might be useful for the upper table in particular if you want to make your own build

Also after giving it some thought, due to the low quality I'd prefer to have my name removed from the metadata

**Comment by archiif** [comment]
> To see the entries that didn't come from jmdict, look at the upper table here

**Comment by archiif** [comment]
> To see the entries that didn't come from jmdict, look at the upper table here

Isn't the upper table the old version which contains the old version of jmdict's entries?

**Comment by Thermospore** [comment]
> Isn't the upper table the old version which contains the old version of jmdict's entries?

No, KireiCake is unrelated. The upper table has the older entries and the lower table has the newer entries. At a glance there is a bit of overlap, but you can think of them as separate volumes.

I think the version of KireiCake on the Yomichan info page is actually an [import](https://github.com/FooSoft/yomichan-import) of the [Rikaichan dictionary](https://www.polarcloud.com/getrcx/), which is how the old JMdict entries got mixed in there? Haven't looked into it tbh

**Comment by FooSoft** [comment]
That's exactly what it is.

**Comment by archiif** [comment]
I see, that makes sense.
Btw, can you provide some instructions on how to build the dictionary? Would be useful if you want to update it yourself.

**Comment by Thermospore** [comment]
@archiif Sure, [here is a video I slapped together](https://youtu.be/fC78dfDbn8M) (Alaskan internet; give it 30 mins to finish uploading)

**Comment by grospoulpe** [comment]
Seems KireiCake is still useful, because without enabling it in my list (JMdict (French), JMdict (English), 研究社　新和英大辞典　第５版, and JMnedict installed, too), やはり is not found. (found in Jisho.org, though: https://jisho.org/word/矢張り )
Sentences used (Japanese, The Manga Way, 247)
なぜ逃げる？**やはり**お前が犯人なんだな！！　　うるせえ！！
Why do you run away? You really are the true killer, aren't you?!　　Get lost!

**Comment by Thermospore** [comment]
Maybe I'm not understanding, but やはり seems to be present in almost all of my dictionaries

And the KireiCake/RikaiCake entry is lackluster

![image](https://user-images.githubusercontent.com/35350819/119271261-4cf82800-bbad-11eb-9e01-327d85776d2a.png)

some don't put the kanji in the headword
![image](https://user-images.githubusercontent.com/35350819/119271152-c93e3b80-bbac-11eb-95cb-5c3bf94b0d42.png)

**Comment by grospoulpe** [comment]
Mmh, I don't have all that stuff.

Maybe something is wrong in my configuration, but I have all JP-EN dictionary downloadable from https://foosoft.net/projects/yomichan/

[yomichan-settings-2021-05-23-20-32-51.zip](https://github.com/FooSoft/yomichan/files/6528886/yomichan-settings-2021-05-23-20-32-51.zip)

**Comment by toasted-nutbread** [comment]
やはり is present in term_bank_28.json of jmdict_english.zip, importing file and enabling the dictionaries worked for me. Perhaps your imported dictionaries are out of date and should be reimported? Not sure what the issue would be otherwise.

**Comment by grospoulpe** [comment]
Thanks, I downloaded that one from https://foosoft.net/projects/yomichan/, and noticed a size difference (it's bigger: 7,555KB vs 6766KB) from the one I downloaded in January 2021. So, I deleted the one installed, and imported the one I've just downloaded.
Now, the term is present.
Thanks again for your response. :)
